tuf/download.py

```diff
@@ -172,20 +163,6 @@ def unsafe_download(url, required_length):
   securesystemslib.formats.URL_SCHEMA.check_match(url)
   securesystemslib.formats.LENGTH_SCHEMA.check_match(required_length)
 
-  # Ensure 'url' specifies one of the URI schemes in
```

Comment:
We probably should retain a comment explaining that this is where the http / https prefix from the settings.py file is checked (assuming that is the case) and that an ???Exception may be thrown as a result...